install/kubernetes/helm/istio/charts/tracing/templates/service.yaml

```diff
@@ -35,7 +35,7 @@ items:
       - name: query-http
         port: 80
```

Comment:
Should this have a property in values? i.e. `.Values.service.externalUiPort`?
Or possibly rename both to be `.Values.service.zipkin.port` and `.Values.service.tracing.port`

Reply:
Good point, I think we can handle this in another PR. For this PR, I want we focus on do not expose containerPort/targePort.